recipes/locust/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "locust" %}
+{% set version = "0.8" %}
+{% set sha256 = "d0f2cf5c979d1c72f5f93cc5d7626e2fbe0c11031b22227370f7c7f7eeb4f5f3" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  noarch: python
+  number: 0
+  script: python -m pip install --no-deps --ignore-installed .
+  entry_points:
+    - locust = locust.main:main
+
+requirements:
+  build:
+    - python
+    - pip
+  run:
+    - python
+    - gevent >=1.2.2
+    - flask >=0.10.1
+    - requests >=2.9.1
+    - msgpack-python >=0.4.2
+    - six >=1.10.0
+    - pyzmq >=16.0.2
+
+test:
+  commands:
+    - locust --help
+
+about:
+  home: https://locust.io/
+  license: MIT
+  license_family: MIT
+  license_file: '{{ environ["RECIPE_DIR"] }}/LICENSE.txt'
```

Comment:
Could you raise an issue (or put in a pull request) asking that the license be included in the source distribution? Then copy the url to the issue or pull request as a comment in the recipe as a reminder that in future updates to the feedstock, the license file might be included in the source.

Reply:
Just to clarify, could you put that link as a comment in the actual recipe `meta.yaml` file above the `license_file` line

Reply:
oh sorry I didn't read your comment carefully enough.... it should be done now